m3/reporter.go

```diff
@@ -288,6 +289,11 @@ func NewReporter(opts Options) (Reporter, error) {
 	internalTags := map[string]string{
 		"version": tally.Version,
 	}
+
+	for k, v := range opts.CommonTagsInternal {
```

Comment:
can you ensure we don't set host/instance here too please (i think we might be inheriting those from common tags)